Add setup guide for debian docker stretch Nvidia harware decode (#234)
jellyfin/jellyfin-docs: general/administration/hardware-acceleration.md

OPTIONAL: After install you may want to add nvidia as default runtime: editing /etc/docker/daemon.json like this:
```
{
    "default-runtime": "nvidia",
    "runtimes": {
        "nvidia": {
            "path": "nvidia-container-runtime",
            "runtimeArgs": []
        }
    }
}
```
Restart docker services
```sudo pkill -SIGHUP docker```

Install nvidia drivers and dependencies:
```apt-get install -t stretch-backports nvidia-driver libnvcuvid1 libnvidia-encode1 libcuda1 nvidia-smi```

Reboot your host:
```reboot now```

Validate your driver and docker are correctly setted up
Driver test from host and docker side:
```
nvidia-smi
docker run --gpus 0 nvidia/cuda:9.0-base nvidia-smi
```
Validate access to needed ressources from host and docker:
```
ldconfig -p | grep cuvid
ldconfig -p | grep libnvidia-encode.so.1
```
Start your Jellyfin container adding thoses environement parameters:
```
-e "NVIDIA_DRIVER_CAPABILITIES=all" \
-e NVIDIA_VISIBLE_DEVICES=all \
--runtime=nvidia \
--gpus all \
```
A complete run command would looks like:
```
docker run -d \
--name=jellyfin \
-e PUID=1000 \

> -e PGID=100 \

Reviewer: Can you run id in your container? AFAIK, the jellyfin image doesn't support user environmental variables.

Author: My bad - this can be removed.
Was usefull when using linuxserver.io container.

Reviewer: so if you want to run the user in the container use --user 1000:1000

you might have to add the user to the video group? depends on your os and who owns the gfx cards. for my rpi4, i had to do 

`usermod -aG video user`

Author: This was not needed on my side to add the user to any group
As the env. settings for the user was not correctly set i guess it was running as root so i had no problem.
Can add infos about this.

Reviewer: sure. any additional info you can add will always be welcome. I dont have an nvidia card so I rely on others for the documentation :)

      # RPi 4
      - /dev/vchiq:/dev/qchiq
```

> ## Debian Docker Nvidia Hardware acceleration:

Reviewer: ```suggestion
## Debian Docker Nvidia:
```

> In order to achieve Hardware decode inside docker containers, some steps are needed:

Reviewer: ```suggestion
In order to achieve Hardware acceleration using docker, some steps are needed:
```

Author: corrected

Thoses steps where wrote for Debian Stretch.

> Prerequisits:

Reviewer: ```suggestion
Prerequisites:
```

https://github.com/nvidia/nvidia-docker/wiki/Installation-(version-2.0)

    GNU/Linux x86_64 with kernel version > 3.10
    Docker >= 1.12
    NVIDIA GPU with Architecture > Fermi (2.1)

>     NVIDIA drivers ~= 361.93 (untested on older versions)

Reviewer: Just curious, have you tried these drivers? 

https://github.com/keylase/nvidia-patch

Author: Yes i did patched my drivers: looks to be working as, if understood the limitation of consumer GPUS, i could get 3 transcodes in parallel
